script/globals.d.ts

```diff
@@ -28,3 +28,37 @@ type LegalEagle = (
   options: LegalEagleOptions,
   callback: (error: Error | null, summary: LicenseLookup) => void
 ) => void
+
+// type declarations for electron-installer-redhat
+type RedhatOptions = {
+  src: string
+  dest: string
+  arch: string
+}
+
+type ElectronInstallerRedhat = (
```

Comment:
You can put a `declare module 'electron-installer-redhat'` block here and put an `export =` statement inside it to automatically use these types when importing `electron-installer-redhat`.

Reply:
I had a bash at getting this in, but couldn't seem to get it right. Feel free to clean up after this PR gets merged (or target a PR onto this branch).